examples/lock-app/nrfconnect/README.md

```diff
@@ -83,6 +90,26 @@ sending the Thread network credentials from the CHIP controller to the CHIP
 device. As a result, device is able to join the Thread network and communicate
 with other Thread devices in the network.
 
+### Device Firmware Upgrade
```

Comment:
If you follow my comments in lighting-app README, then align it also in lock-app README